### PR TITLE
feat: authentication added

### PR DESCRIPTION
**Pull request by NandhiniKrishnamoorthy**
Added authentication page, Please Review

Preview Link: https://sonafancy-login.netlify.app/

### CODE REVIEW

**Review by JowelTisso — APPROVED** [review]
Suggestion: You can remove the commented code

File: authentication/login.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <!-- font awesome icons -->
+    <!-- <script src="https://kit.fontawesome.com/c2fdf537fc.js"
+            crossorigin="anonymous"></script> -->
+    <title>sona fancy</title>
```
Comment (dev-enforced) [inline authentication/login.html:12]: Remove these comments

File: authentication/login.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                        <i class="fas fa-shopping-cart icon"></i>
+                        <span class="badge-icon badge-red">2</span>
+                    </div>
+                </a>
+                <a href="../authentication/login.html">
+                    <div class="ml-1 badge-element cursor-pointer">
+                        <i class="fas fa-user-circle icon"></i>
+                    </div>
+                </a>
+                <!-- <button class="button-main primary-button mx-2 mr-2 cursor-pointer">Login</button> -->
```
Comment (dev-enforced) [inline authentication/login.html:57]: Remove these comments

**Review by dev-enforced — APPROVED** [review]
Code looks good to me and the demonstration also looks good

**Pull request merged**